### PR TITLE
create reviews controller

### PR DESCRIPTION
**Pull request by ilariaperoni**
I have created a Reviews controller with the `create` action and private `review_params` method

### CODE REVIEW

**Comment by fransan6** [comment]
Need to have a discussion about the link of review to booking or of review to venue. TBC

File: test/controllers/reviews_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require "test_helper"
```
Comment (jondedman) [inline test/controllers/reviews_controller_test.rb:1]: What is the test helper?

Comment (fransan6) [reply test/controllers/reviews_controller_test.rb:1]: I believe this file and the test_helper is automatically generated when you create a controller

**Review by alexagodzilla — COMMENTED** [review]
LGTM 
Let's merge it, there are no conflicts with other pieces of code but please double check line 8 on app/controllers/reviews_controller.rb

**Pull request merged**